src/main/resources/intentionDescriptions/JoinWildcardsIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention converts successive <code>_</code> wildcard(s) to <code>..</code>.
```

Comment:
```suggestion
This intention converts successive <code>_</code> wildcard(s) to <code>..</code>
```
Oh, this dot at the end can be confusing

Reply:
![image](https://user-images.githubusercontent.com/3221931/126657059-0d8c1cd2-d6f5-4eaf-a854-87bc51601c76.png)

Reply:
Hmm, I wonder if anyone even reads these intention descriptions :D

Reply:
(me too, but anyway)

Reply:
JFYI I suppose it will fail on `RsIntentionTestBase.test intention has documentation`